Source/dom/components/__tests__/ImportableObject.test.js

```diff
@@ -0,0 +1,43 @@
+/* globals expect, test */
+import { isRunningOnJenkins } from '../../../test-utils'
+import { Library, Document, Artboard, Text, SymbolMaster } from '../../'
+
+// some tests cannot really run on jenkins because it doesn't have access to MSDocument
+if (!isRunningOnJenkins()) {
+  let lib
+
+  test('should import a symbol from a lib', () => {
+    const document = new Document()
+
+    const artboard = new Artboard({
+      name: 'Test',
+      parent: document.selectedPage,
+    })
+    // eslint-disable-next-line
+    const text = new Text({
+      text: 'Test value',
+      parent: artboard,
+    })
+    // eslint-disable-next-line
+    const master = SymbolMaster.fromArtboard(artboard)
+    lib = Library.createLibraryFromDocument(
+      document,
+      '~/Desktop/sketch-api-unit-tests-importable-objects.sketch'
+    )
+
+    const document2 = new Document()
+
+    const symbolRefs = lib.getImportableSymbolReferencesForDocument(document2)
+
+    expect(symbolRefs.length).toBe(1)
+    expect(symbolRefs[0].id).toBe(master.symbolId)
+
+    const importedMaster = symbolRefs[0].importInDocument(document2)
```

Comment:
Just a thought here. If the user did:
```
refs = lib.getImportableSymbolReferencesForDocument(document)
master = refs[0].importInDocument(differentDocument)
```
That could potentially leave things in a bad state (e.g. if the symbol was already in `document` and not in `differentDocument` we not do anything and trying to create an instance would result in a missing symbol.

Would it be worth protecting against this by storing the target document on the `ImportableObject`? So instead we could just do:
```
refs = lib.getImportableSymbolReferencesForDocument(document)
master = refs[0].import()
```

Reply:
Good idea